### PR TITLE
refactor: use table instead of a list to display user tokens

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
na

https://github.com/gravitee-io/gravitee-api-management/pull/290

**Description**

Prefer use of mat table to display tokens
before :
image

after
image

**Additional context**

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-pcxrgibvkr.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/user-details-tokens/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts [hunk 1]
```diff
@@ -109,6 +116,9 @@ export class OrgSettingsUserDetailComponent implements OnInit, OnDestroy {
 
   tokens: Token[];
```
Comment (ytvnr) [inline gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts]: I think you can delete this line ?

**Review by ytvnr — APPROVED** [review]
just a remark

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts (hunk 1 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts]: 🚀

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-qwewcfflcp.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/user-details-tokens/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.spec.ts [hunk 2]
```diff
@@ -662,13 +659,11 @@ describe('OrgSettingsUserDetailComponent', () => {
     ]);
 
     const tokensCard = await loader.getHarness(MatCardHarness.with({ selector: '.org-settings-user-detail__tokens__card' }));
-    const tokensList = await tokensCard.getHarness(MatListHarness);
-    const matListItemHarnesses = await tokensList.getItems();
-    expect(matListItemHarnesses).toHaveLength(1);
+    const tokensTable = await tokensCard.getHarness(MatTableHarness);
+    expect(await tokensTable.getCellTextByIndex()).toEqual([['My token', 'Aug 31, 2021, 3:35:35 AM', 'Sep 7, 2021, 2:18:25 PM', 'delete']]);
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.spec.ts]: Where does this date comes from, `fakeUserToken`? maybe we should add them in the attribute to make the link more clear

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-qydfcwqdgc.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/user-details-tokens/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-xnckezendj.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/user-details-tokens/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-vukcvwnjmq.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/user-details-tokens/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-abnrxqxnxc.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/user-details-tokens/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-dckfighflg.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-teagcmsmsj.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/user-details-tokens/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-hnztclejsg.chromatic.com)

**Review by ytvnr — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/user-details-tokens/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Pull request merged**